packages/lib-sourcify/src/lib/types.ts

```diff
@@ -108,7 +108,9 @@ export interface Metadata {
     };
     evmVersion: string;
     libraries?: {
-      [index: string]: string;
+      [index: string]: {
+        [index: string]: string;
+      };
```

Comment:
Are we sure about this? The Solidity docs (which I wrote) also confirms the prev. type https://docs.soliditylang.org/en/v0.8.20/metadata.html